debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java

```diff
@@ -419,7 +419,7 @@ protected void handleServerHeartbeat(MySqlPartition partition, MySqlOffsetContex
      */
     protected void handleServerIncident(MySqlOffsetContext offsetContext, Event event) {
         if (event.getData() instanceof EventDataDeserializationExceptionData) {
-            metrics.onErroneousEvent("source = " + event.toString());
+            metrics.onErroneousEvent("source = " + event, null);
```

Comment:
Looking at the number of places where this method needs to be invoked with `null` now, I'd suggest to have two overloaded versions of that method: with and without the operation parameter.

Reply:
fix these issue